src/compiler/parser.ts

```diff
@@ -1189,26 +1189,30 @@ namespace ts {
             return node;
         }
 
-        function createMissingNode<T extends Node>(kind: T["kind"], reportAtCurrentPosition: boolean, diagnosticMessage: DiagnosticMessage, arg0?: any): T {
-            if (reportAtCurrentPosition) {
-                parseErrorAtPosition(scanner.getStartPos(), 0, diagnosticMessage, arg0);
-            }
-            else {
-                parseErrorAtCurrentToken(diagnosticMessage, arg0);
-            }
-
+        function createMissingNodePlaceholder<T extends Node>(kind: T["kind"]) {

```

Comment:
We don't need a new kind of missing node to handle this one case. Please work with the existing infrastructure for missing nodes

Reply:
updated 😄